ci-scripts/publish.sh

```diff
@@ -31,5 +31,7 @@ npm publish
 
 cd ../..
 
+npm install
+
```

Comment:
I think this can be left out — I don’t think this was an issue with `npm install` getting called, it was that it was never defined as a dependency.